eknrenderer/eknr-boxed-set-info.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2018 Endless Mobile, Inc. */
+
+#include "eknr-boxed-set-info.h"
+
+typedef struct _EknrBoxedSetInfoReal {
+  EknrBoxedSetInfo base;
+  guint            ref_count;
+} EknrBoxedSetInfoReal;
+
+EknrBoxedSetInfo *
+eknr_boxed_set_info_new (gboolean             featured,
+                         const char          *ekn_id,
+                         const char          *title,
+                         GStrv                child_tags,
+                         GStrv                tags)
+{
+  EknrBoxedSetInfo *set = (EknrBoxedSetInfo *) g_new0 (EknrBoxedSetInfoReal, 1);
+
+  ((EknrBoxedSetInfoReal *) set)->ref_count = 1;
+  set->featured = featured;
+  set->ekn_id = g_strdup (ekn_id);
+  set->title = g_strdup (title);
+  set->child_tags = g_strdupv ((GStrv) child_tags);
+  set->tags = g_strdupv ((GStrv) tags);
+
+  return set;
+}
+
+EknrBoxedSetInfo *
+eknr_boxed_set_info_ref (EknrBoxedSetInfo *set)
+{
+  ++((EknrBoxedSetInfoReal *) set)->ref_count;
```

Comment:
Should the ref count not use `g_atomic_int_inc()` and `g_atomic_int_dec_and_test()`?

Reply:
It probably should, yeah.